hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -720,7 +723,11 @@ public PendingKeysDeletion getPendingDeletionKeys(
       String volume, String bucket, String startKey,
       CheckedFunction<KeyValue<String, OmKeyInfo>, Boolean, IOException> filter,
       int count) throws IOException {
+    boolean isLegacy = ozoneManager.getScmInfo().getMetaDataLayoutVersion() <
```

Comment:
isLegacy -> isDataDistributionFinalized or is DataDistributionEnabled. 

Legacy is not welly used in Ozone.  There are continuous new features are implemented, it's better to use a more meaningful and straightforward naming, to a better readability.